commands/alias.go

```diff
@@ -36,7 +36,7 @@ func alias(command *Command, args *Args) {
 	}
 
 	if shell == "" {
-		utils.Check(fmt.Errorf("Unknown shell"))
+		utils.Check(fmt.Errorf("Unknown shell: Set the SHELL enviornment variable or execute 'hub alias -s <shell name>'"))
```

Comment:
Let's not suggest that people set the SHELL environment variable, and instead let's just suggest that they explicitly pass in the shell type as argument.

How about if the error message said:

> Error: couldn't detect shell type. Please specify your shell with `hub alias -s <shell>`

Also, note that this error message can be shown when `hub alias` was called without the `-s` flag. In that case, the error messsage *shouldn't* suggest that you execute `hub alias -s`. Instead:

> Error: couldn't detect shell type. Please specify your shell with `hub alias <shell>`